### ISSUE TITLE
iefolder: allow globals pointing at different directories to allow integration with git/cloud service (eg.dropbox)

### ISSUE THREAD

**Issue by jonasguthoff** [worldbank/ietoolkit#196]
Allow for customization or an option where one could specify that do files are stored in a different directory than the data, so that globals are set accordingly and point at the correct directory. 
This would provide an easy setup for teams that keep data stored in a cloud service and the code on github.

**Comment by kbjarkefur** [comment]
Thanks @jonasguthoff ! I will look into this!

**Comment by bbdaniels** [comment]
This could allow two other useful setups:
1. Making sure the file structures remain parallel (ie, setting up in parallel on first run, then checking for a `parallel` flag on run to add new rounds/folders and executing the same in the other location automatically); and
2. Calling `iegitaddmd` automatically for the Git location

**Comment by kbjarkefur** [comment]
Yes, when we do this we should consider all features we can add around Git!
